### ISSUE TITLE
Expected line break before `.callsFake`  newline-per-chained-call

### ISSUE THREAD

**Issue by shellac85** [typescript-eslint/typescript-eslint#2189]
**Repro**

```JSON
{
  "rules": {
    "@typescript-eslint/tslint/config": ["warn", {
      "rules": {
        "rule": "setting",
      },
      "rulesDirectory": [
        "node_modules/foo"
      ]
    }],
  }
}
```

```TS
// your repro code case
```

**Expected Result**
Code looks fine
**Actual Result**
Getting an error when I run npm run lint -- --fix regarding 
newline-per-chained-call
**Additional Info**
Please see below. 

**Versions**

| package                                   | version |
| ----------------------------------------- | ------- |
| `@typescript-eslint/eslint-plugin-tslint` | `1.7.0` |
| `@typescript-eslint/parser`               | `1.7.0` |
| `TypeScript`                              | `3.4.5` |
| `ESLint`                                  | `5.16.0` |
| `node`                                    | `10.17.0` |
| `npm`                                     | `6.11.3` |

Hi. I have this code below which is triggering an error: 
239:24  error  Expected line break before `.callsFake`  newline-per-chained-call

Can anyone see what the issue is here or have a solution? Tks
```
      fetchSysAidStub
        .onFirstCall().callsFake(() => Promise.resolve(companiesResponse))
        .onSecondCall().callsFake(() => Promise.resolve(countResponse))
        .onThirdCall().callsFake(() => Promise.resolve(ticketsResponse));
```

**Comment by bradzacher** [comment]
Could you please provide your config? It's hard to help you with a config issue without any config.

Help me to help you.

**Comment by shellac85** [comment]
Sorry is it the eslintrc.js contents you are looking for?

**Comment by bradzacher** [comment]
yes

**Comment by shellac85** [comment]
```
module.exports = {
    env: {
      browser: true,
      es6: true,
      node: true,
      mocha: true,
    },
    extends:  [
      'airbnb-base',
      'plugin:@typescript-eslint/recommended',
    ],
    globals: {
      Atomics: 'readonly',
      SharedArrayBuffer: 'readonly',
      ENV_PRODUCTION: 'readonly',
    },
    parser: '@typescript-eslint/parser',
    parserOptions: {
      ecmaVersion: 2018,
      project: './tsconfig.json',
      sourceType: 'module',
    },
    plugins: [
      '@typescript-eslint',
      'mocha',
    ],
    rules: {
      'quotes': ["error", "single"],
      'no-console': 0,
      'no-unused-vars': 0,
      'no-return-await': 0,
      'indent': [2, 2],
      'linebreak-style': 0,
      'comma-dangle': [
        "error",
        {
          "arrays": "always-multiline",
          "objects": "always-multiline",
          "imports": "always-multiline",
          "exports": "always-multiline",
          "functions": "always-multiline"
        }
      ],
      'class-methods-use-this': 0,
      'lines-between-class-members': 0,
      'object-curly-newline': 0,
      'import/no-extraneous-dependencies': ['error', {'devDependencies': ['**/*.test.*', '**/*.spec.*', 'buildconfig/**']}],
      'import/prefer-default-export': 0,
      '@typescript-eslint/indent': 0,
      '@typescript-eslint/no-object-literal-type-assertion': 2,
      '@typescript-eslint/explicit-function-return-type': 0,
      '@typescript-eslint/no-empty-interface': 1,
      '@typescript-eslint/no-unused-vars': ['error', { vars: 'all', args: 'after-used', ignoreRestSiblings: true, argsIgnorePattern: '^_'}],
      'mocha/no-exclusive-tests': 'error',
    },
    settings:  {
      "import/resolver": {
        typescript: {},
        node: {
          paths: ["src", "types"],
          extensions: ['.js', '.ts'],
        }
      }
    },
  };

```

**Comment by bradzacher** [comment]
you're using the `airbnb-base` config, which enables the `newline-per-chained-call` rule.

The documentation for the lint rule should help explain what the rule checks and how to fix it:
https://eslint.org/docs/rules/newline-per-chained-call

**Comment by shellac85** [comment]
Ok thanks, but each call is on a new line

```
 fetchSysAidStub
        .onFirstCall().callsFake(() => Promise.resolve(companiesResponse))
        .onSecondCall().callsFake(() => Promise.resolve(countResponse))
        .onThirdCall().callsFake(() => Promise.resolve(ticketsResponse));
```

**Comment by bradzacher** [comment]
they're not though - `.onFirstCall().callsFake(` is two separate calls.

**Comment by shellac85** [comment]
oh lord. How did I miss that. Thank you for your help man. Error is gone!
